src/language/en-US.json

```diff
@@ -186,6 +186,12 @@
   "transmittable": "Transferable",
   "variable_supply": "Supply mutable",
   "duration_permanent": "Unlimited duration",
+  "flags_supply": "The total supply must be in the range of 0 and 9,000,000,000,000,000 atomic units.",
+  "flags_divisibility": "Determines the decimal place to which the mosaic can be divided.The divisibility must be in the range of 0 and 6.",
+  "flags_transmittable": "If set to true,the mosaic can be transferred between arbitrary accounts.",
```

Comment:
- space after comma
- key should be `flags_transferable`.
- `If set to true, the mosaic can be transferred to arbitrary accounts.`